app/views/trainees/funding/bursaries/_tiered_bursary_form.html.erb

```diff
@@ -0,0 +1,36 @@
+<%= register_form_with(model: @bursary_form, url: trainee_funding_bursary_path(@trainee), method: :put, local: true) do |f| %>
+  <%= f.govuk_error_summary %>
+
+  <h1 class="govuk-heading-l"><%= t("components.page_titles.trainees.funding.bursaries.edit") %><h1>
+  <p class="govuk-body">
+    <%= t("views.forms.funding.bursaries.tiered.description") %>
+    </br>
+    </br>
+    <%= t("views.forms.funding.bursaries.guidance_html") %>
+  </p>
+
+  <%= f.govuk_radio_buttons_fieldset(:bursary_tier, legend: { text: t("views.forms.funding.bursaries.title"), size: "m" }) do %>
+
+    <%= f.hidden_field :applying_for_bursary, value: false %>
+
+    <% BURSARY_TIERS.each do |tier_key, _tier_value| %>
+      <%= f.govuk_radio_button(
+        :bursary_tier,
+        tier_key,
+        label: { text: t("views.forms.funding.bursaries.tiered.#{tier_key}.label") },
+        hint: { text: t("views.forms.funding.bursaries.tiered.#{tier_key}.hint") },
+        link_errors: true,
+      ) %>
+    <% end %>
+
+    <%= f.govuk_radio_divider %>
+
+      <%= f.govuk_radio_button(
+        :bursary_tier,
+        "",
```

Comment:
When I first arrive on this for, the "No do not apply for a bursary" radio button is pre-selected. This shouldn't happen.

<img width="693" alt="Screenshot 2021-07-07 at 14 10 55" src="https://user-images.githubusercontent.com/18436946/124764805-40494c80-df2d-11eb-94a6-3ce0b4a02516.png">